assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -114,6 +123,11 @@ export function QuickTip({
               <source src={`${cdnURL}${figureSrc}.mp4`} type="video/mp4" />
             </Video>
           )}
+          {/* `figureSrcImg` is temporary until we get an animation for the embed quick tip.
+          Once we have the animation then the .png image won't be necessary */}
+          {figureSrcImg && (
+            <Image alt={figureAlt} src={`${cdnURL}${figureSrcImg}.png`} />
```

Comment:
If you want, you could just do `Video as='img'` to reuse the style since it's duplicative.